tests/integration/mounts_sshfs.bats

```diff
@@ -31,16 +31,16 @@ function setup_sshfs() {
 		-o StrictHostKeyChecking=no
 		-o PasswordAuthentication=no"
 
-	if ! [ -v DIR ]; then
-		DIR="$BATS_RUN_TMPDIR/fuse-sshfs"
-		mkdir -p "$DIR"
-		# Make sure we clear all superblock flags to make sure bind-mounts can
-		# unset these flags.
-		if ! $sshfs -o rw,suid,dev,exec,atime rootless@localhost: "$DIR"; then
-			# fallback to tmpfs if running in without sshfs
-			mount -t tmpfs -o rw,suid,dev,exec,diratime,strictatime tmpfs "$DIR"
-		fi
+	DIR="$BATS_RUN_TMPDIR/fuse-sshfs"
+	umount "$DIR" 2>/dev/null || true
```

Comment:
This umount and doing the first mount unconditionally is what makes the trick

Reply:
Yeah, as I suspected in https://github.com/opencontainers/runc/issues/4093#issuecomment-2005170113, doing a `umount` and then a new mount would ensure that all of the mount flags are cleared properly.

The only explanation I can think of is that Debian's mount is different to everyone else's util-linux `mount`. And I think this might actually be the case -- [it seems Debian doesn't use util-linux's `mount` at all](https://sources.debian.org/src/mount/2.7l-5/mount.c/), instead they use a really old `mount` program that seems to have very different behaviour to libmount.

Reply:
It doesn't seem to be the case, as I mentioned there :(

Reply:
Found the issue, closing this. Please feel free to re-open if I'm missing something